src/main/java/com/bina/varsim/fastqLiftover/readers/ARTFastqAlnReader.java

```diff
@@ -30,7 +30,7 @@ public SimulatedRead getNextRead() throws IOException {
             return null;
         }
         if (nameLine.trim().length() < 1) {
-            log.warning("got empty string at FASTQ line " + fastqBr.getLineNumber());
+            log.severe("got empty name string at FASTQ line " + fastqBr.getLineNumber());
```

Comment:
Looks like you are using java.util.logger instead of log4j's logger. Which is why you do not have `error(...)`